watermark/watermark.py

```diff
@@ -340,7 +339,7 @@ def _get_conda_env():
 def _get_gpu_info():
     if py3nvml is None:
         return {"GPU Info": "Install the gpu extra "
-                "(pip install 'watermark[gpu]') "
+                "(pip install watermark[gpu]) "
```

Comment:
Thanks for the PR! I think this may not work on certain terminal shells (bash I think) and require the comment characters. Are the comment characters cause any issues on your terminal, which is why you removed them, or was this more of a cosmetic change?

Reply:
> Are the comment characters cause any issues on your terminal, which is why you removed them, or was this more of a cosmetic change?

Did/Does not work for me in command shell, only when removing the `'` characters:
![grafik](https://github.com/user-attachments/assets/a0147193-ceee-4046-83d5-db35c275c91c)

Reply:
Huh, wow. What happens if you use the double-quote character: `pip install "watermark[gpu]"`?

Reply:
Good idea, looks better 🙂 
![grafik](https://github.com/user-attachments/assets/7476c0b2-ba13-4f87-8c36-c3ade4766283)

Reply:
Oh nice. Just updated the line in this PR accordingly. Would that work for you?

Reply:
lgtm!

Reply:
@rasbt Ah, there is small typo, there is a wrong `)` after `[gpu]`. It should be
`'(pip install "watermark[gpu]") '`
not
`'(pip install "watermark[gpu])") '`